core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/actions/resource/CreateLabel.java

```diff
@@ -0,0 +1,80 @@
+package de.valtech.aecu.core.groovy.console.bindings.actions.resource;
+
+import de.valtech.aecu.api.service.AecuException;
+import de.valtech.aecu.core.groovy.console.bindings.actions.Action;
+import org.apache.sling.api.resource.PersistenceException;
+import org.apache.sling.api.resource.Resource;
+
+import javax.annotation.Nonnull;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+/**
+ *@author Ian Smets
+ *@author Wietse Vandeput
+**/
+public class CreateLabel implements Action {
+    private final String language;
+    private final String key;
+    private final String value;
+    String PROPERTY_SLING_MESSAGE = "sling:message";
```

Comment:
make constants `private static final` and put them above class variables

Reply:
constants updated and moved